subsys/ipc/ipc_service/ipc_service.c

```diff
@@ -12,6 +12,11 @@
 #include <zephyr.h>
 #include <device.h>
 
+#if defined(CONFIG_MBOX_INIT_PRIORITY)
+BUILD_ASSERT(CONFIG_IPC_SERVICE_REG_BACKEND_PRIORITY > CONFIG_MBOX_INIT_PRIORITY,
+		"IPC service backend must be initialized after MBOX driver.");
+#endif
+
```

Comment:
This should not be here because not necessarily the backend is using the MBOX. This is true for `static_vrings` but it's not a general solution.

Reply:
Ah, yes. Sorry i forgot to move it.

Reply:
Discussed offline. This assertion is not required for any backend right now. Im removing the last commit.